CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## Update manifests
+
+Envoy use a Jinja2 template to store its `envoy.yaml` and applies during its deployments. This is stored in `src/templates`. Envoy-related manifests are part of the KFP manifests. The process for updating them is:
+
+### Spot the differences between versions of manifest files
+
+1. Install `kustomize` using the official documentation [instructions](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+2. Clone [Kubeflow manifests](https://github.com/kubeflow/manifests) repo locally
+3. `cd` into the repo and checkout to the branch or tag of the target version.
+4. Build the manifests with `kustomize` according to instructions in https://github.com/kubeflow/manifests?tab=readme-ov-file#kubeflow-pipelines.
```

Comment:
AFAIK the manifests for metadata envoy are not rendered from this command. Could you please confirm the source is correct?